xwiki-pro-macros-confluence-bridges/xwiki-pro-macros-confluence-bridges-ui/src/main/resources/Confluence/Macros/ConfluenceDetailsSummary.xml

```diff
@@ -320,7 +320,7 @@ Bridges are there for compatibility with content migrated from Confluence and t
             #set ($d = $xwiki.getDocument($row.get(0)))
           #end##
           #foreach ($cell in $row)
-            |#if ($foreach.first)[[$cell]]#{else}$services.rendering.escape($cell, $xwiki.currentContentSyntaxId)#end##
+            | ((( #if ($foreach.first)[[$cell]]#{else}$cell#end ))) ##
```

Comment:
We are wrapping all the children of the table cell in a div to ensure that we don't generate new blocks that might break the UI.